features/edit_delete_task.feature

```diff
@@ -27,28 +27,6 @@ Feature: Edit & Delete Task
     And I should see "Maximum budget: 700"
     And I should see "Category: Computer"
 
-  Scenario: Unable to edit another requester's task
-    Given I am logged in as "requester1@r.com"
-    And I visit the landing page
-    And I click on "View My Tasks"
-    And I click link "Bad computer"
-    And I click on "Edit task"
-    And I fill in field "Task Name" with "Broken computer"
-    And I fill in field "Description" with "faulty powerpack"
-    And I select "Computer" from "Task Category"
-    And I fill in field "Minimum Budget:" with "500"
-    And I fill in field "Maximum Budget:" with "700"
-    And I fill in field "Location" with "Gothenbourg"
-    And I click on "Submit"
-    Then I should see "Task Updated!"
-    And I should see "Task Name: Broken computer"
-    And I should see "Description: faulty powerpack"
-    And I should see "Location: Gothenbourg"
-    And I should see "Minimun budget: 500"
-    And I should see "Maximum budget: 700"
-    And I should see "Category: Computer"
-
```

Comment:
Because the current functionality we have in the code doesn't allow a requester to see another requester's tasks so it wasn't needed. This would be used in the new feature working with CanCanCan